### DISCUSSION TITLE
guide on using external DB?

### DISCUSSION THREAD

**Discussion by cbc02009** [babybuddy/babybuddy#509] [Q&A] [answered]
I would like to set up babybuddy to use an external Mariadb instance I already have available. the linuxserver.io page says to use the environment variables found in https://github.com/babybuddy/babybuddy#configuration, but the link is dead. I've tried looking at the new docs here: https://docs.baby-buddy.net/configuration/database/ but I'm completely clueless as to what they mean.

Could anyone provide an example or some guidance on how to achieve this?

**Comment by cdubz** [comment, marked as answer]
Sorry about the dead link. I just broke that documentation in to subsections.

I don't have an example for you (perhaps someone else using LSIO may be able to cook one up) but essentially you need to pass those environment variables to the app to use them.

The primary (and probably most confusing) on you'll need for you situation is `DB_ENGINE` which you can set to `django.db.backends.mysql` for MariaDB support.

The rest of the variables should be pretty self-explanatory (host, name, user, password, port).

**Reply by cbc02009** [reply]
Thanks, I think the engine explanation was exactly what I was missing. I'll give it a try!

**Reply by cdubz** [reply]
Great. Let me know if you run in to any issues. I haven't done extensive testing of databases other than PostgreSQL and SQLite so curious to know if Maria/MySQL works out of the box with LSIO images...
